hack/lib/vars.bash

```diff
@@ -5,7 +5,7 @@ readonly BUILD_NUMBER=${BUILD_NUMBER:-$(uuidgen)}
 # shellcheck disable=SC1091,SC1090
 source "$(dirname "${BASH_SOURCE[0]}")/../../test/vendor/github.com/knative/test-infra/scripts/e2e-tests.sh"
 
-readonly KNATIVE_VERSION="${KNATIVE_VERSION:-v0.13.1}"
+readonly KNATIVE_VERSION="${KNATIVE_VERSION:-v0.13.2}"
```

Comment:
shouldn't this be named `KNATIVE_SERVING` ? Since the the serverless operator installs now both: serving/eventing

Reply:
presumably, yeah.